plasmapy/physics/transport/collisions.py

```diff
@@ -306,11 +309,22 @@ def _boilerPlate(T, particles, V):
     # obtaining reduced mass of 2 particle collision system
     reduced_mass = atomic.reduced_mass(*particles)
 
-    if V == 0:
+    if np.any(V == 0):
         raise utils.exceptions.PhysicsError("You cannot have a collision for zero velocity!")
     # getting thermal velocity of system if no velocity is given
-    if np.isnan(V):
+    if V is None:
```

Comment:
Won't this need a change in the function's signature? We might also then have to add `V: {'none_shall_pass': True}` to the `check_quantity` dictionary.

Reply:
Hmm, good point, I'm not sure.  The current decorator situation seems to allow me pass a None in as the V input, I think I did that in one of the tests. 

My rationale for including this check was to allow for either None or np.nan * unit to indicate "no input specified, use the default behavior". I think there are parts of the code right now where we've used either one or the other as the argument default.

Reply:
Yeah, the `np.nan` thing was a hack to pass in a `None`-ish `Quantity. I'll look at it a bit later.

Reply:
Weird, previously passing `None` would always fail unit checks from the decorator...